interfaces/HO-Portal/src/app/program/program.component.html

```diff
@@ -36,12 +36,12 @@
       </ion-col>
     </ion-row>
     <ion-row>
-      <ion-col size="auto">
+      <ion-col size=3>
```

Comment:
This doesn't need to be committed together with the other changes. :|

Reply:
Shall I remove it?